public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.ts

```diff
@@ -245,10 +245,10 @@ function getElements(state: DashboardSceneState) {
   return elements;
 }
 
-function getPanelLinks(panel: VizPanel): DashboardLink[] {
+function getPanelLinks(panel: VizPanel): DataLink[] {
   const vizLinks = dashboardSceneGraph.getPanelLinks(panel);
   if (vizLinks) {
-    return (vizLinks.state.rawLinks as DashboardLink[]) ?? [];
+    return (vizLinks.state.rawLinks as DataLink[]) ?? [];
```

Comment:
This type assertion ain't needed here i think. Also, rawLink is expressed as a runtime data link which may contain more properties (the runtime one) that we need to get rid of.

Reply:
You were right, no type assertion needed, I've updated this. The runtime data link has additional properties, but they are not persisted and relevant only to explore. It's this one right? [DataLink](https://github.com/grafana/grafana/blob/002f872ce17fe9d93a949008223be9170983c333/packages/grafana-data/src/types/dataLink.ts#L32)